ee/api/test/test_team.py

```diff
@@ -0,0 +1,50 @@
+from unittest.mock import patch
+
+from django.test.utils import tag
+
+from posthog.models.organization import Organization, OrganizationMembership
+from posthog.models.team import Team
+from posthog.models.user import User
+
+from .base import APILicensedTest
+
+
+class TestTeamEnterpriseAPI(APILicensedTest):
+    def test_create_team(self):
+        response = self.client.post("/api/projects/", {"name": "Test"})
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(Team.objects.count(), 2)
+        response_data = response.json()
+        self.assertEqual(response_data.get("name"), "Default")
+        self.assertEqual(self.organization.teams.count(), 2)
+
+    def test_delete_team_own_second(self):
+        team = Team.objects.create(organization=self.organization)
+        response = self.client.delete(f"/api/projects/{team.id}")
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(Team.objects.filter(organization=self.organization).count(), 1)
+
+    def test_no_delete_last_team(self):
```

Comment:
Discussed this with @timgl. Having last org/project be deletable would invalidate a lot of our current assumptions, because it's always been assumed that `User.team` is never `None`. For example in such cases most sidebar items would have to be hidden, endpoints would have to be disabled. I think we should make this flexible in the future, although I'd preface that with #1775.

Reply:
Sounds good 👌